libbeat/outputs/mode/balance.go

```diff
@@ -282,19 +279,13 @@ func (m *LoadBalancerMode) forwardEvent(
 	msg eventsMessage,
 ) bool {
 	if msg.attemptsLeft < 0 {
-		select {
-		case ch <- msg:
-			return true
-		case <-m.done: // shutdown
-			return false
-		}
+		ch <- msg
+		return true
```

Comment:
Actually it should have the same behavior with or without ```case <-done``` because here we make sure to output all events received. 
The publisher being closed, balance ```forwardEvent``` will not be called after we reach WGEvent.Wait()